solidus_reviews.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |spec|
   spec.executables = files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'solidus_core', ['>= 2.0.0', '< 3']
+  spec.add_dependency 'solidus_core', ['>= 2.7.0', '< 4']
```

Comment:
Why the Solidus `2.0` -> `2.7` bump as well?

Reply:
That's an error. Good catch. Fixed now.